task/yq/0.1/yq.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: yq
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.24.1"
```

Comment:
is there anything specific in this task which makes it incompatible with versions lower than `0.24.1` of pipelines?

Reply:
Looks like I could do 0.12.1 based on comparision with yaml lint example.